service/common/src/main/java/org/apache/polaris/service/catalog/iceberg/CatalogHandlerUtils.java

```diff
@@ -94,6 +98,18 @@
 public class CatalogHandlerUtils {
   private static final Schema EMPTY_SCHEMA = new Schema();
   private static final String INITIAL_PAGE_TOKEN = "";
+  private static final String CONFLICT_RESOLUTION_ACTION =
+      "polaris.conflict-resolution.by-operation-type.replace";
```

Comment:
I think we should try to stick to the convention `polaris.XYZ.etc` where `XYZ` is like the subtype of property. So far we have `polaris.config` for the Polaris configs. 

IIUC this is only set/read by Polaris. Maybe `polaris.internal`?